src/elements/content-sidebar/activity-feed/comment/BaseComment.js

```diff
@@ -168,133 +173,147 @@ const BaseComment = (props: Props) => {
 
     return (
         // TODO: Change className to bcs-Comment once FF is removed
-        <div className="bcs-BaseComment">
-            <Media
-                className={classNames('bcs-Comment-media', {
-                    'bcs-is-pending': isPending || error,
-                })}
-            >
-                <Media.Figure>
-                    <Avatar getAvatarUrl={getAvatarUrl} user={createdByUser} />
-                </Media.Figure>
-                <Media.Body>
-                    {isMenuVisible && (
-                        <TetherComponent
-                            attachment="top right"
-                            className="bcs-Comment-deleteConfirmationModal"
-                            constraints={[{ to: 'scrollParent', attachment: 'together' }]}
-                            targetAttachment="bottom right"
-                        >
-                            <Media.Menu
-                                isDisabled={isConfirmingDelete}
-                                data-testid="comment-actions-menu"
-                                dropdownProps={{
-                                    onMenuOpen: handleMenuOpen,
-                                    onMenuClose: handleMenuClose,
-                                }}
-                                menuProps={{
-                                    'data-resin-component': ACTIVITY_TARGETS.COMMENT_OPTIONS,
-                                }}
+        <div className={classNames('bcs-BaseComment', { reply: !!parentID, parent: !parentID })}>
+            <div className={classNames('bcs-BaseComment-selectWrapper', { reply: !!parentID, parent: !parentID })}>
```

Comment:
SUIT CSS prefers descendent naming, so "wrapper" is atypical. Would this be `bcs-BaseComment-content`?